src/features/notice/ui/NoticeList/NoticeList.tsx

```
@@ -0,0 +1,236 @@
+			return '높음';
+		case 'medium':
+			return '보통';
+		case 'low':
+			return '낮음';
+		default:
+			return priority;
+	}
+};
+
+// 드래그 가능한 공지사항 아이템 컴포넌트
+const SortableNoticeItem = ({ notice, onEditNotice, onDelete }: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)